cue_file

```diff
@@ -280,19 +270,16 @@ result = analyse(
 )
 
 # prepare string-based JSON result
-# 2 decimals is exact enough, also some tools don't understand 3 or more decimals
 # we use "dB" instead of "LU" units, because LS & others don’t understand "LU"
 liq_result = {
-    "duration": f"{result['duration']:.2f}",
-    "liq_duration": f"{result['liq_duration']:.2f}",
-    "liq_cue_in": f"{result['liq_cue_in']:.2f}",
-    "liq_cue_out": f"{result['liq_cue_out']:.2f}",
-    "liq_cross_start_next": f"{result['liq_cross_start_next']:.2f}",
-    "liq_longtail": "true" if (result["liq_longtail"]) else "false",
-    "liq_cross_duration": f"{result['liq_cross_duration']:.2f}",
+    "duration": result['duration'],
```

Comment:
I'm not sure I see the benefit of converting floats to string and then back to float when reading the JSON payload.